server/server.js

```diff
@@ -36,12 +41,15 @@ class Server {
     app.set('views', `${__dirname}/views`);
     app.set('view engine', 'ejs');
 
+    app.use(Raven.requestHandler());
     // compresses the content in gzip
     app.use(compression());
     app.use(bodyPaser());
     app.use('/static', express.static(`${__dirname}/../dist`));
     app.use(routes);
 
+    app.use(Raven.errorHandler());
```

Comment:
should only collect errors in production enviroment.

Reply:
fixed